life_expectancy/loading_saving.py

```diff
@@ -1,29 +1,43 @@
+"""
+Module to load and save data
+"""
+
 from pathlib import Path
 
 import pandas as pd
 
+PARENT_PATH = Path(__file__).parent
+FILE_PATH = PARENT_PATH / 'data'
+TSV_FILE_NAME = 'eu_life_expectancy_raw.tsv'
+JSON_FILE_NAME = 'eurostat_life_expect.json'
+TSV_FILE_PATH = FILE_PATH / TSV_FILE_NAME
+JSON_FILE_PATH = FILE_PATH / JSON_FILE_NAME
 
-def load_data(file_path: str) -> pd.DataFrame:
-    """
-    Method used to load data.
-
-    Args:
-        file_path [str]: file path to load data.
 
-    Returns:
-        [pd.DataFrame]: Dataframe to be cleaned.
+class DataLoader:
+    """
+    This class is responsible for verifying the format
+    and assign the right load method
     """
-    data_df = pd.read_csv(file_path, sep='\t')
+    def __init__(self, data_format):
+        self.data_format = data_format
+
+    def load_data(self) -> None:
+        """Method to verify data format"""
+        if self.data_format == 'csv':
+            return pd.read_csv(TSV_FILE_PATH, sep='\t')  # load csv file
+        if self.data_format == 'json':
+            return pd.read_json(JSON_FILE_PATH)  # load json file
```

Comment:
If you have a file path attribute, you can get rid of the duplicate constants:

```suggestion
        if self.data_format == '.csv':
            return pd.read_csv(self.file_path, sep='\t')
        if self.data_format == '.json':
            return pd.read_json(self.file_path)
```

Reply:
Got it! Thank you!